ironic/manifest.jsonnet

```diff
@@ -0,0 +1,249 @@
+local kpm = import "kpm.libjsonnet";
+
+function(
+  params={}
+)
+
+kpm.package({
+  package: {
+    name: "stackanetes/ironic",
+    expander: "jinja2",
+    author: "Mateusz Blaszkowski",
+    version: "0.1.0",
+    description: "ironic",
+    license: "Apache 2.0",
+  },
+
+  variables: {
+    deployment: {
+      control_node_label: "openstack-control-plane",
+      compute_node_label: "openstack-ironic-compute-node",
+      replicas: 1,
+
+      image: {
+        base: "quay.io/stackanetes/stackanetes-%s:barcelona",
+
+        init: $.variables.deployment.image.base % "kolla-toolbox",
+        post: $.variables.deployment.image.base % "kolla-toolbox",
+        # TODO:
+        api: "10.91.96.87:5000/debian-source-ironic-api:mateuszb-ironic",
+        db_sync: "10.91.96.87:5000/debian-source-ironic-api:mateuszb-ironic",
+        conductor: "10.91.96.87:5000/debian-source-ironic-conductor:mateuszb-ironic",
+        compute: "10.91.96.87:5000/debian-source-nova-compute-ironic:mateuszb-ironic",
+        ironic_pxe: "10.91.96.87:5000/debian-source-ironic-pxe:mateuszb-ironic",
+        ironic_http: "nginx:alpine",
+      },
+    },
+
+    network: {
+      ip_address: "{{ .IP }}",
+
+      port: {
+        api: 6385,
+        http: 8082,
+      },
+
+      api_url: "http://%s:6385/v1" % $.variables.network.ip_address,
+
+      dns:  {
+        servers: ["10.3.0.10"],
+        kubernetes_domain: "cluster.local",
+        other_domains: "",
+      },
+
+      ingress: {
+        enabled: true,
+        host: "%s.openstack.cluster",
```

Comment:
Values from the manifest are overwritten with the same values from `parameters.yaml` (if present). However, they **need** to be specified in a manifest.